lib/StateMachineSkill.js

```diff
@@ -43,12 +35,18 @@ class StateMachineSkill extends AlexaSkill {
 
     _.each(requestToTurnIntoIntent, (requestToConvert) => {
       this[requestToConvert]((alexaEvent, reply) => {
-        const intent = _.chain(requestToConvert)
-          .split('')
-          .drop(2)
-          .join('')
-          .replace('.', '')
-          .value();
+        let intent;
+
+        if (requestToConvert === 'onLaunchRequest') {
+          intent = this.config.openIntent || 'LaunchIntent';
+        } else {
+          intent = _.chain(requestToConvert)
```

Comment:
this is also something hard to ready, maybe we could split it into a separate function? Like, `requestNameToIntent` or something like that is easier to understand than
```js
            .split('')
            .drop(2)
            .join('')
            .replace('.', '')
            .value();
```

We could also have a test that shows `onPlaybackController.PlayCommandIssued` is transformed into `PlaybackControllerPlayCommandIssuedIntent` will make us understand why the need for all those 'replace', drop, etc

Reply:
Done!